### PR TITLE
Redo tutorial localization and popup locations

### LINKED ISSUE #350: Update tutorial for Beta 1 changes
* Facility and infil tutorials are outdated
* The covert actions one could use re-phrasing (currently it sound like heavier gear reduces infil time). Also need to better explain that normal CAs without ambush risk will not result in a mission
* The geoscape one could use a rework to account for activity chains spawning

### LINKED ISSUE #355: No tutorial for Assault missions
We should probably tell the player that assaults are more difficult than infils

### PR DESCRIPTION
**Pull request by cannonfodder1**
Closes #350 
Closes #355

### CODE REVIEW

File: CovertInfiltration/Src/CovertInfiltration/Classes/UICovertActionsGeoscape_FactionHeader.uc [hunk 1]
```diff
@@ -77,7 +77,7 @@ simulated protected function UpdateText()
 
 	if (bIsOngoing)
 	{
-		strText = class'UIUtilities_Infiltration'.static.MakeFirstCharCapOnly(class'UICovertActions'.default.CovertActions_CurrentActiveHeader);
```
Comment (Xymanek) [inline CovertInfiltration/Src/CovertInfiltration/Classes/UICovertActionsGeoscape_FactionHeader.uc]: Can you explain this change?

Comment (cannonfodder1) [reply CovertInfiltration/Src/CovertInfiltration/Classes/UICovertActionsGeoscape_FactionHeader.uc]: Before, it was `In progress` which was really annoying. Now it's `In Progress`

File: CovertInfiltration/Localization/CovertInfiltration.int [hunk 2]
```diff
@@ -341,25 +342,37 @@ strLabel=View Chain
...
 
-strFacilityChangesHeader="Facility Changes"
-strFacilityChangesBody="The Resistance Ring is no longer needed to prepare Covert Actions - instead, building it unlocks Resistance Orders. Once the Ring is built, you will be able to assign orders instantly, without waiting for a supply drop.<br/>You are now limited in how many personnel you can maintain at once by the Barracks Limit. For each soldier, scientist, or engineer over the limit, will recovery times are increased by 10%. The limit can be increased via upgrades in the Living Quarters.<br/>In addition, the GTS now has 2 rookie training slots."
+strFacilityGTSHeader="Guerilla Tactics School Changes"
+strFacilityGTSBody="The Guerilla Tactics School has undergone an overhaul. It now has 2 training slots, and instead of training Rookies to Squaddies it now has a Training Rank value. Any soldier under the rank can be staffed to train up to the Training Rank. It starts at Squaddie but can be increased to Corporal and Sergeant through Resistance Orders and a GTS Unlock."
```
Comment (Xymanek) [inline CovertInfiltration/Localization/CovertInfiltration.int]: When training rookies, you are still able to select their future class

File: CovertInfiltration/Localization/XComGame.int [hunk 3]
```diff
@@ -273,12 +273,12 @@ CovertActionsBuildRing="SELECT OPERATION"
 [LivingQuarters_BarracksSizeI X2FacilityUpgradeTemplate]
 DisplayName="Barracks Size I"
 FacilityName="Living Quarters"
-Summary="Increases Barracks capacity by 10, allowing more soldiers to be maintained without an increase to recovery times."
+Summary="Increases crew capacity by 10, allowing more personnel to be maintained without an increase to recovery times."
```
Comment (Xymanek) [inline CovertInfiltration/Localization/XComGame.int]: I don't see the actual resource bar label renamed

File: CovertInfiltration/Localization/CovertInfiltration.int [hunk 4]
```diff
@@ -340,26 +341,38 @@ strCounterHiddenDarkEventDescription="Resistance heard about a <font color='#E52
...
+strGeoscapeEntryHeader=Proactive Deployment
+strGeoscapeEntryBody=In Covert Infiltration, most missions will no longer appear automatically on the geoscape. Instead, opportunities will appear in the Covert Ops menu, accessible from the button at the bottom of the screen. You can send out as many squads as you wish but be aware that new opportunities may appear at any time and that ADVENT can and will initiate retaliations against your havens!
 
-strCovertActionLoadoutHeader="Changes to Covert Actions"
-strCovertActionLoadoutBody="Traditional Covert Actions still involve you sending out soldiers into the unknown, but there are a few major changes. First, the time to complete them is now dependent on your soldiers' gear. A properly equipped squad will complete them more quickly.<br/>Secondly, you also have to watch out for the squad's deterrence, which affects the trigger chance of risks. Deterrence increased by sending more experienced soldiers and giving them better gear.<br/>Balance out infiltration and deterrence to complete Covert Actions successfully!"
+strBarracksLimitHeader=Barracks Limit
```
Comment (Xymanek) [inline CovertInfiltration/Localization/CovertInfiltration.int]: Plz rename to "crew" (+ the resource header label)

**Pull request merged**